### DISCUSSION TITLE
Can target be higher than es6?

### DISCUSSION THREAD

**Discussion by antoniormrzz** [typegoose/typegoose#438] [Q&A] [answered]
I'm interested in using typegoose with typegraphql, typegraphql insists on a higher target than es6. your docs doesn't say anything other than es6, a bit confusing. I'm talking about tsconfig target.

**Comment by hasezoey** [comment, marked as answer]
sorry if the docs are not clear, `es6` is the least required target, this is mentioned because `es5` is still typescript default

**Comment by antoniormrzz** [comment]
Thanks for the quick reply, I could do a PR for docs and readme if you need 😃
